lib/clevertap_plugin.dart

```diff
@@ -426,17 +426,24 @@ class CleverTapPlugin {
         'setLocation', {'latitude': latitude, 'longitude': longitude});
   }
 
+  @Deprecated("This method is deprecated since v1.2.4. Use getCleverTapID() instead")
```

Comment:
Should be `This method is deprecated since v1.3.0. Use getCleverTapID() instead` @william-ct